sdk/python/kfp_tekton/compiler/_k8s_helper.py

```diff
@@ -115,3 +115,90 @@ def convert_k8s_obj_to_json(k8s_obj):
 
     return {key: convert_k8s_obj_to_json(val)
             for key, val in iteritems(obj_dict)}
+
+
+def _to_str(s):
+    return None if s is None else str(s)
```

Comment:
Thanks @jizg, quick question. Is there a reason we need to check for `None` here? Doesn't this get checked in sanitize_k8s_object? Or are there plans to use `_to_str ()` outside this file in the future?

Reply:
You are right, '_to_str()' is not necessary now. I firstly did the 'None' type check in '_to_str()', and then moved the check to `sanitize_k8s_object` but forgot to remove '_to_str()'. I have updated the codes. Thanks.